### ISSUE TITLE
window.mixpanel.opt_out_tracking() is not a function

### ISSUE THREAD

**Issue by rstudner** [mixpanel/mixpanel-js#189]
I can't seem to actually call this function ever?

Has it been removed or changed but the docs aren't updated?

**Comment by tdumitrescu** [comment]
Nope, nothing's changed, it's still there: https://github.com/mixpanel/mixpanel-js/blob/4a107013408bc743dbc23aeb9ecbe79c5054ee52/src/mixpanel-core.js#L1732-L1764

You can contact support@mixpanel.com to get help with debugging the details of your particular implementation.
